docker/cpu/Dockerfile

```diff
@@ -39,4 +42,4 @@ ENV no_proxy=$no_proxy
 
 RUN pip install setuptools
 
-ENTRYPOINT cd /home/nncf/nn-compression-framework/ && python setup.py install --cpu-only  && bash
+ENTRYPOINT cd /home/nncf/ && python setup.py develop --cpu-only && bash
```

Comment:
What was wrong with the 'install' setup type?

Reply:
It was nothing wrong actually. I've just needed to test it for sure and forget to change back. This is fixed in next commit.